tests/robottelo/test_decorators.py

```diff
@@ -106,7 +106,8 @@ class MockBug(object):  # pylint:disable=R0903
             whiteboard = None
         dec_settings.upstream = False
         decorators._get_bugzilla_bug = lambda bug_id: MockBug()
-        for MockBug.status in BZ_OPEN_STATUSES + BZ_CLOSED_STATUSES:
+        # skip 'CLOSED' state, as it overides whiteboard 'verified in upstream'
```

Comment:
Also make the test ensure that 'CLOSED' is the last item on the list. That comment does not have a reason why it is needed and someone may change that. If the test ensure that, then it will start failing and will not introduce any false positive here on the test.